### ISSUE TITLE
Command onlyoffice-desktopeditors --new:cell does not open a new spreadsheet

### ISSUE THREAD

**Issue by joshcangit** [ONLYOFFICE/DesktopEditors#145]
**Do you want to request a *feature* or report a *bug*?**
This is a bug.

**What is the current behavior?**
The desktop file uses command `onlyoffice-desktopeditors --new:cell`, in [issue #124 ](https://github.com/ONLYOFFICE/DesktopEditors/issues/124), to open a New Spreadsheet.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
![newcell](https://user-images.githubusercontent.com/33022160/47766208-79818700-dcc5-11e8-8ad9-2c90bc23807d.gif)

**What is the expected behavior?**
Click on New Spreadsheet
![newcell](https://user-images.githubusercontent.com/33022160/47766611-7be4e080-dcc7-11e8-8bbe-22f52cf2bf00.png)
New Spreadsheet created
![book1](https://user-images.githubusercontent.com/33022160/47766624-974feb80-dcc7-11e8-8960-af144ae354e1.png)

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
```
snap    2.35.4
snapd   2.35.4
series  16
ubuntu  18.04
```
```
Name                       Version  Rev  Tracking  Publisher    Notes
onlyoffice-desktopeditors  5.1.27   26   stable    onlyoffice✓  -
```

**Comment by Rita-Bubnova** [comment]
Hello, @joshcangit.
Thank you. I can confirm - this is a bug, issue 39523 in our internal issue tracker.

**Comment by joshcangit** [comment]
I can confirm this is also fixed in Version 5.2.4, latest deb package from the official repository as of today.
No more using the Snap version. It's just not as updated nor is it as snappy in the way it worked.
It also can't open files from certain folders inside my home directory such as the one I synced to Google Drive. This is even after I gave it permission to access files in my home folder and read/write files to removable storage devices.
